.clj-kondo/hooks/toucan/util/test.clj

```diff
@@ -5,7 +5,9 @@
   (let [pairs       (partition 2 bindings)
         db-refs     (map first pairs)
         let-stream  (for [[_ binding+opts] pairs
-                          part             (:children binding+opts)]
+                          ;; if children is size 1 then ensure nil is on the right side of the let
+                          :let [[binding-part opts] (:children binding+opts)]
+                          part [binding-part opts]]
```

Comment:
this fixes bad analysis when `with-temp*` doesn't use a pair of values for the model.

```
(mt/with-temp [Action [a] HttpAction [b {}]] ...)
```

This was expanding into `[[Action HttpAction] (let [a b {}] ...)]` but should now turn into `[[Action HttpAction] (let [a nil b {}] ...)]`